RetailCoder.VBE/Refactorings/ImplementInterface/ImplementInterfaceRefactoring.cs

```diff
@@ -87,7 +87,7 @@ public void Refactor(QualifiedSelection selection)
 
             if (oldSelection.HasValue)
             {
-                using (var module = oldSelection.Value.QualifiedName.Component.CodeModule)
+                var module = _state.ProjectsProvider.CodeModule(oldSelection.Value.QualifiedName);
```

Comment:
Shouldn't the `module` be disposed too?

Reply:
No, the disposal of things on the repository is handled by the repository.